<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Elpida::RunTaskBatchDialog</class>
 <widget class="QDialog" name="Elpida::RunTaskBatchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run Task Batches</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/BG-tile.svg)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="gbTaskBatches">
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="title">
        <string>Available Task Batches</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListWidget" name="lwTaskBatches"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbResults">
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="title">
        <string>Results</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTreeWidget" name="twResults">
          <property name="styleSheet">
           <string notr="true">background: rgb(240,244,255)</string>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Result</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="bgRunDetails">
     <property name="styleSheet">
      <string notr="true">background:transparent;</string>
     </property>
     <property name="title">
      <string>Run Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="lblCurrentTaskBatch">
        <property name="text">
         <string>Current Task Batch:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lblCurrentTaskName">
        <property name="text">
         <string>N/A</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblCurrentTask">
        <property name="text">
         <string>Current Task:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lblCurrentTaskBatchName">
        <property name="text">
         <string>N/A</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblStatus">
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblSatusValue">
        <property name="text">
         <string>&lt;span style=&quot;color:#008d09;&quot;&gt;Ready&lt;/span&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="taskBatchRunProgress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlControls">
     <item>
      <widget class="QPushButton" name="pbRun">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbStop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hsControls">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btnbDialogControls">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnbDialogControls</sender>
   <signal>accepted()</signal>
   <receiver>Elpida::RunTaskBatchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnbDialogControls</sender>
   <signal>rejected()</signal>
   <receiver>Elpida::RunTaskBatchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
